ingestion/src/metadata/ingestion/source/pipeline/airflow.py

```diff
@@ -320,19 +320,34 @@ def yield_pipeline(
         :return: Create Pipeline request with tasks
         """
 
-        dag: SerializedDAG = self._build_dag(pipeline_details.data)
-        yield CreatePipelineRequest(
-            name=pipeline_details.dag_id,
-            description=dag.description,
-            pipelineUrl=f"/tree?dag_id={dag.dag_id}",  # Just the suffix
-            concurrency=dag.concurrency,
-            pipelineLocation=pipeline_details.fileloc,
-            startDate=dag.start_date.isoformat() if dag.start_date else None,
-            tasks=self.get_tasks_from_dag(dag),
-            service=EntityReference(
-                id=self.context.pipeline_service.id.__root__, type="pipelineService"
-            ),
-        )
+        try:
+            dag: SerializedDAG = self._build_dag(pipeline_details.data)
+            yield CreatePipelineRequest(
+                name=pipeline_details.dag_id,
+                description=dag.description,
+                pipelineUrl=f"/tree?dag_id={dag.dag_id}",  # Just the suffix
+                concurrency=dag.concurrency,
+                pipelineLocation=pipeline_details.fileloc,
+                startDate=dag.start_date.isoformat() if dag.start_date else None,
+                tasks=self.get_tasks_from_dag(dag),
+                service=EntityReference(
+                    id=self.context.pipeline_service.id.__root__, type="pipelineService"
+                ),
+            )
+        except TypeError as err:
+            logger.error(
+                f"Error building DAG information from {pipeline_details}. There might be Airflow version"
+                f" incompatibilities - {err}"
+            )
+            logger.debug(traceback.format_exc())
```

Comment:
nit: in other parts of the code, we display debug before error

Reply:
thanks for the ping, pushing the change